app/controllers/agents_controller.rb

```diff
@@ -287,4 +325,53 @@ def find_agent_display_all(id = params[:id])
       obj.id.to_s.eql?("#{rest_url}/Agents/#{id}")
     end.first
   end
+
+  def normalize_orcid(orcid)
```

Comment:
too much complex, do `orcid.split('/').last` no need to validate the number.

Reply:
We need to keep orcid in this form (to stay concistent with what we have):
1234-1234-1234-1234 with the dashes between numbers,
this function is like a black box, all u need to know is the input: orcid and the output: normalized orcid or nil if not valid